CellFabric/Cell_Fabric_FinFET__Mock/Align_primitives.py

```diff
@@ -48,9 +38,9 @@ def main( args):
                 SB.append(lS)
                 GB.append(lG)
                 DB.append(lD)
-    ### End ###
-    
-    if args.primitive in ["Switch_NMOS", "Switch_PMOS", "DCL_NMOS", "DCL_PMOS"]:
+    elif any(args.primitive.startswith(f'{x}_') for x in ["Switch", "DCL"]):
+        # What pattern are we trying to do here?
```

Comment:
Not quite clear what pattern we are implementing here. The pattern switch is definitely ignored.

Reply:
For single type transistors  (e.g., switch, Diode connected load, etc.) we don't need a pattern. All transistors are part of the same device.

Reply:
Okay thank you.